jdaviz/tests/test_subsets.py

```diff
@@ -135,12 +133,10 @@ def test_region_from_subset_3d(cubeviz_helper):
 
 
 def test_region_from_subset_profile(cubeviz_helper, spectral_cube_wcs):
-    data = Data(flux=np.ones((128, 128, 256)), label='Test 1D Flux', coords=spectral_cube_wcs)
```

Comment:
For the glue display axis to initialize properly, we cannot hack-load like this anymore.